wordpress-mobile/AztecEditor-iOS
# Fix caption processor

#879 

This PR make sures the Caption processor returns caption align with the image attributes. For example: 

```
[caption id="attachment_2816" align="alignnone" width="3000"]
<img class="alignnone wp-image-2816" src="https://sergioestevaosales.files.wordpress.com/2018/03/img_0003-8.jpg" width="3000" height="2002" /> 
Waterfall
[/caption]
```

To test:

- Open the demo app
- Open the editor demo with content
- Insert an image
- Add a caption to the image
- Switch to HTML
- Check if the attributes in caption are correct.


Keep in mind these changes are needed in order to show captions correctly on the WordPress web editors only.
